docs/core/tools/using-ci-with-cli.md

```diff
@@ -78,12 +78,12 @@ for more information.
 
 ### AppVeyor
 
-The [appveyor.com ci](https://www.appveyor.com/) has .NET Core SDK preview2 already installed in the build worker image `Visual Studio 2015`.
+The [appveyor.com ci](https://www.appveyor.com/) has .NET Core SDK 1.0.1 already installed in the build worker image `Visual Studio 2017`.
```

Comment:
this one is another one that needs a more extensive revision and is covered here: https://github.com/dotnet/docs/pull/1556